app/views/layouts/_header.html.erb

```diff
@@ -244,8 +244,16 @@
                       class: "category-header--nav-item #{active?(current_cat) && !['tags', 'suggested_edit'].include?(controller_name) ? 'is-active' : ''}" %>
           <%= link_to 'Tags', category_tags_path(current_cat),
                       class: "category-header--nav-item #{active?(current_cat) && controller_name == 'tags' ? 'is-active' : ''}" %>
-          <%= link_to 'Edits', suggested_edits_queue_path(current_cat),
-                      class: "category-header--nav-item #{active?(current_cat) && controller_name == 'suggested_edit' ? 'is-active' : ''}" %>
+          <%= link_to suggested_edits_queue_path(current_cat),
+                      class: "category-header--nav-item #{active?(current_cat) && controller_name == 'suggested_edit' ? 'is-active' : ''}" do %>
+            Edits
+            <% has_pending = Rails.cache.fetch "pending_suggestions/#{current_cat.id}" do %>
+              <% SuggestedEdit.where(post: Post.undeleted.where(category: current_cat), active: true).any? %>
+            <% end %>
```

Comment:
suggestion: This is controller-type code in a view - should probably move to a helper that can be called here instead.